File: GestionAeropuerto.API/Models/Reserva.cs
namespace GestionAeropuerto.API.Models
{
    public class Reserva
    {
        public int ReservaID { get; set; }
        public string CodigoReserva { get; set; } = string.Empty;
        public int VueloID { get; set; }
        public int PasajeroID { get; set; }
        public string NumeroAsiento { get; set; } = string.Empty;
        public string Clase { get; set; } = "Economica";
        public string Estado { get; set; } = "Confirmada";
        public decimal Precio { get; set; }
        public int Equipaje { get; set; } = 1;
        public DateTime FechaReserva { get; set; }

        public string? NombrePasajero { get; set; }
        public string? NumeroVuelo { get; set; }
   
    }
}

File: GestionAeropuerto.VB/Reserva.vb
Public Class Reserva
    Public Property ReservaID As Integer
    Public Property CodigoReserva As String
    Public Property VueloID As Integer
    Public Property NumeroVuelo As String
    Public Property PasajeroID As Integer
    Public Property NombrePasajero As String
    Public Property NumeroAsiento As String
    Public Property Clase As String
    Public Property Estado As String
    Public Property Precio As Decimal
    Public Property Equipaje As Integer
    Public Property FechaReserva As DateTime

    Public Sub New()
        CodigoReserva = ""
        NumeroVuelo = ""
        NombrePasajero = ""
        NumeroAsiento = ""
        Clase = "Economica"
        Estado = "Confirmada"
        Equipaje = 1
    End Sub
End Class